PORT-3736 itemsToParse & plain text algorithm docs (#333)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/webhook/webhook.md

}
```

> - The `itemsToParse` key let you to create multiple entities from a single webhook event:

Reviewer: ```suggestion
- The `itemsToParse` makes it possible to create multiple entities from a single webhook event:
```

> - The `itemsToParse` makes it possible to create multiple entities from a single webhook event:

Reviewer: ```suggestion
- The `itemsToParse` key makes it possible to create multiple entities from a single webhook event:
```

> :::note

Reviewer: Move the note under the JSON example

- Any JQ expression can be used here, as long as it evaluates to an array of items.

> - `item` will be added to the JQ context as a key containing the current item being processed and can be accessed using the `.item`.

Reviewer: ```suggestion
- `item` will be added to the JQ context as a key containing a reference to items in the array specified in `itemsToParse`. Keys from the object in the array can be accessed using the `.item.KEY_NAME` syntax, see the example JSON for more information.
```

>   :::

Reviewer: This indent broke the formatting

### Security configuration

> When 3rd party services send payload to a specified webhook URL, they will also include a header containing the signature of the payload.

Reviewer: ```suggestion
When 3rd party services send payload to a specified webhook URL, they will usually also include a header containing a signed signature of the payload, or some agreed upon string for verification of the sender.
```

The signature might be generated by running a SHA-X (for example SHA-1 or SHA-256) hashing function on the payload, combined with a secret value specified by the user or provided by the 3rd party service at the time of webhook creation.

> The security option "plain" is available for use. With this option, the signature is compared to the secret value without any modification. It allows users to bypass the hashing function and directly compare the signature to the provided secret value. This can be useful in scenarios where a simpler security mechanism is desired.

Reviewer: ```suggestion
Since some 3rd party service do not offer sending the payload signature, and instead only offer sending an agreed upon string directly - the security option `plain` is available for use. With this option, the signature is compared to the secret value without any modification. It allows users to directly compare the signature to the provided secret value. This can be useful in scenarios where a simpler security mechanism is desired.
```

}
```

> :::note

Reviewer: Remove this note, the :::info below contains almost exactly the same text and they are very close to each other, so we can remove one

}
```

:::info

> When using the `plain` algorithm, it will use the secret to match the exact signature header without trying to decrypt it.

Reviewer: ```suggestion
When using the `plain` algorithm, no hashing will be performed and the value of the secret saved in the Port webhook configuration will be compared to the value in the specified header without any modification.
```
